testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/cli/TestRun.java

```diff
@@ -203,6 +203,7 @@ public Execution(EnvironmentFactory environmentFactory, EnvironmentOptions envir
             if (impactedFeaturesFile.isPresent()) {
                 try {
                     this.impactedFeatures = Optional.of(Files.asCharSource(impactedFeaturesFile.get(), StandardCharsets.UTF_8).readLines());
+                    log.info("Impacted features: %s", this.impactedFeatures);
```

Comment:
Doesn't hurt to print it here but we already print this in the CI: https://github.com/trinodb/trino/blob/master/.github/workflows/ci.yml#L809